go.mod

```diff
@@ -30,15 +32,14 @@ require (
 	github.com/kitex-contrib/tracer-opentracing v0.0.3
 	github.com/opentracing/opentracing-go v1.2.0
 	github.com/prometheus/client_golang v1.13.0 // indirect
-	github.com/sirupsen/logrus v1.9.0 // indirect
+	github.com/sirupsen/logrus v1.9.0
```

Comment:
add some licenses in licenses's dir if  it is direct dependency.

Reply:
done, thx